pkg/controller/servicebindingrequest/olm.go

```diff
@@ -237,7 +235,7 @@ func buildCRDDescriptionFromCRD(crd *unstructured.Unstructured) (*olmv1alpha1.CR
 
 // buildDescriptorsFromAnnotations builds two descriptors collection, one for spec descriptors and
 // another for status descriptors.
-func buildDescriptorsFromAnnotations(annotations map[string]string) (
+func buildDescriptorsFromAnnotations(in map[string]string) (
```

Comment:
why changing param name?

Reply:
Conflicts with `annotations` package name, and I didn't feel `anns` would be a good name for either the package alias or the parameter name (there is another part of this PR that need to be renamed at some point).

Reply:
`annotationMap`?

Reply:
[I don't consider a good practice to having types in names of symbols as well.](https://softwareengineering.stackexchange.com/questions/102689/what-is-the-benefit-of-not-using-hungarian-notation)